test/src/tests.js

```diff
@@ -63,24 +63,59 @@ describe('Bing Ads Event Forwarder', function() {
     beforeEach(function() {
         reportService.reset();
         window.uetq = [];
-        mParticle.forwarder.init(
-            {
-                tagId: 'tagId',
-            },
-            reportService.cb,
-            true
-        );
     });
 
     describe('Init the BingAds SDK', function() {
+        beforeEach(function() {
+            mParticle.forwarder.init(
+                {
+                    tagId: 'tagId',
+                },
+                reportService.cb,
+                true
+            );
+        });
+
         it('should init', function(done) {
             window.uetq.length.should.equal(0);
 
             done();
         });
+
+        it('should init with a consent payload', function(done) {
+            mParticle.forwarder.init(
+                {
+                    tagId: 'tagId',
+                },
+                reportService.cb,
+                false // Disable testMode so we can test init
+            );
+
+            // UETQ queues up events as array elements and then parses them internally.
+            // The first 3 elements of this array will be the consent payload
+            // The 4th element will be the event payload
```

Comment:
This is a bit confusing, but maybe I'm being a bit dense.  You mention `The 4th element`, but there are only 3 elements here.  I don't think it's clear that an event has not yet been logged.  Maybe the comment should mention something about that?  Something along the lines of "Future events will be pushed onto this array", or something like that?  Though I'm not sure we need to bring up an event payload of 4th element at all here.